templates/content-single-question.php

```diff
@@ -2,8 +2,8 @@
 /**
  * Single Question Template
  *
- * @since    1.0.0
- * @version 3.16.0
+ * @since 1.0.0
+ * @version [version]
```

Comment:
changelog please
```suggestion
 * @since 1.0.0
 * @since 3.16.0 Unknown.
 * @since [version] Pass the `$attempt` object when retrieving the question content via `$question->get_question();`
 * @version [version]
```